stats/stats.go

```diff
@@ -60,7 +61,20 @@ func StartStatsServerWithPort(startupOpts StartupOptions, addhandlers ...func(mu
 	}
 
 	var logLevel zap.AtomicLevel
-	if startupOpts.LogLevel != nil {
+	if envLogLevel := os.Getenv("LOG_LEVEL"); envLogLevel != "" {
+		var setLevel zapcore.Level
+		switch envLogLevel {
+		case "debug":
+			setLevel = zapcore.DebugLevel
+		case "warn":
+			setLevel = zapcore.WarnLevel
+		case "error":
+			setLevel = zapcore.ErrorLevel
+		default:
+			setLevel = zapcore.InfoLevel
+		}
+		logLevel = zap.NewAtomicLevelAt(setLevel)
+	} else if startupOpts.LogLevel != nil {
```

Comment:
Another approach for this is passing in the log level from higher up in the stack as a startup option. I decided against that approach since 1) that would still be based on an env variable set through helm and 2) would result in some duplicated code across the different places where log level is configurable.